packages/lib/components/link/link.css

```diff
@@ -0,0 +1,87 @@
+@keyframes waveline {
+  to {
+    background-position: 150px;
+  }
+}
+
+.cx-link {
+  --line-height: 1.5rem;
+  --underline-height: 2px;
+  --underline-squiggle-height: 7px;
+  --underline-offset: calc(
+    var(--line-height) +
+    ((var(--underline-squiggle-height) - var(--underline-height)) / 2)
+  );
+  --underline-squiggle-offset: var(--line-height);
+
+  color: var(--cx-color-text-primary);
+  line-height: var(--line-height);
+  font-size: 1.125rem;
```

Comment:
Skal vanlig link font size være 18px?

Reply:
Hmm men da blir font-size avhengig av font-size lenger opp i DOM treet, og jeg tipper tanken er at font-size skal være fixed til 18px?

Reply:
Vi har ikke brukt lenker noe særlig enda, men tanken var at inline brukes i tekst og arver, mens ellers skal de være fixed når de står for seg selv.